views/templates/albums.html

```diff
@@ -0,0 +1,106 @@
+            <input id="name" data-ng-model="albumsIndexCtrl.newAlbum.name" type="text" class="form-control input-md" required="">
+            </div>
+          </div>
+
+          <!-- Text input-->
+          <div class="form-group">
+            <label class="col-md-4 control-label">Artist Name</label>
+            <div class="col-md-4">
+            <input data-ng-model="albumsIndexCtrl.newAlbum.artistName" type="text" class="form-control input-md">
+            </div>
+          </div>
+
+          <!-- Text input-->
+          <div class="form-group">
+            <label class="col-md-4 control-label">Release Date</label>
+            <div class="col-md-4">
+            <input data-ng-model="albumsIndexCtrl.newAlbum.releaseDate" type="text" class="form-control input-md">
+            </div>
+          </div>
+
+          <!-- Textarea -->
+          <div class="form-group">
+            <label class="col-md-4 control-label">Genres (separated by commas)</label>
+            <div class="col-md-4">
+              <textarea class="form-control" data-ng-model="albumsIndexCtrl.newAlbum.genres"></textarea>
+            </div>
+          </div>
+
+
+          <!-- Button -->
+          <div class="form-group">
+            <label class="col-md-4 control-label">Save Album</label>
+            <div class="col-md-4">
+              <button class="btn btn-primary">Submit</button>
+            </div>
+          </div>
+        </fieldset>
+      </form>
+    </div>
+  </div>
+
+  <input type="text" class="form-control" data-ng-model="searchText" placeholder="Search Albums">
+  <div data-ng-repeat="album in albumsIndexCtrl.albums | orderBy: 'name' | filter: searchText" class="row">
+    <div class="col-md-10 col-md-offset-1">
+      <div class="panel panel-default">
+        <div class="panel-body">
+
+          <div class='row'>
+            <div class="col-md-3 col-xs-12 thumbnail">
+              <img src="images/800x800.png" alt="album image">
+            </div>
+
+            <div class="col-md-9 col-xs-12">
+              <ul class="list-group form-inline">
+                <li class="list-group-item">
+                  <h4 class='inline-header'>Album Name:</h4>
+                  <span data-ng-hide="editing"><a data-ng-href="{{album._id}}">{{album.name}}</a></span>
+                  <input data-ng-show="editing" data-ng-model="album.name" type="text" class="form-control input-md">
+                </li>
+
+                <li class="list-group-item">
+                  <h4 class='inline-header'>Artist Name:</h4>
+                  <span data-ng-hide="editing">{{album.artistName}}</span>
+                  <input data-ng-show="editing" data-ng-model="album.artistName" type="text" class="form-control input-md">
+                </li>
+
+                <li class="list-group-item">
+                  <h4 class='inline-header'>Released date:</h4>
+                  <span data-ng-hide="editing">{{album.releaseDate}}</span>
```

Comment:
We need to go through this and make a list of each built-in directive and make sure it's introduced in the directives lecture.